### PR TITLE
store and retrieve url info

### PR DESCRIPTION
**Pull request by nilrelyt**

### CODE REVIEW

**Comment by thuanvoit** [comment]
Sorry What does this one do? Can you explain a little?

**Review by LordArjuna — APPROVED** [review]

File: src/main/java/iluvus/backend/api/controller/PostController.java
Comment (LordArjuna) [inline src/main/java/iluvus/backend/api/controller/PostController.java:1]: good job tyler

File: src/main/java/iluvus/backend/api/model/Post.java
Comment (LordArjuna) [inline src/main/java/iluvus/backend/api/model/Post.java:1]: adding comments to file can help readability

**Review by LordArjuna — APPROVED** [review]

**Review by thuanvoit — APPROVED** [review]
I think process a list of media urls is good enough.

**Pull request merged**